lib/inspec/input.rb

```diff
@@ -318,6 +318,21 @@ def has_value?
       !current_value.is_a? NO_VALUE_SET
     end
 
+    #--------------------------------------------------------------------------#
+    #                               Marshalling
+    #--------------------------------------------------------------------------#
```

Comment:
I'm all for code comments, but these seem a bit off. Could we make these compatible with rdoc? https://ruby.github.io/rdoc/

Reply:
Whole heartedly agree. This becomes an rdoc/ri comment on the `#to_hash` method.

I also don't think these improve navigability of code/files.

I'd rather we move towards having our public API have proper rdoc/ri comments.